### PR TITLE
Do not display two times the unstable branch

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

File: templates/project/README.md.twig [hunk 1]
```diff
@@ -21,7 +21,9 @@ It's auto-generated by sonata-project/dev-kit package.
 
 Branch | Github Actions | Code Coverage |
 ------ | -------------- | ------------- |
+{% if project.stableBranch is not null %}
 {{ project.stableBranch.name }}    | [![Test][test_stable_badge]][test_stable_link]     | [![Coverage Status][coverage_stable_badge]][coverage_stable_link]     |
```
Comment (OskarStark) [inline templates/project/README.md.twig:24]: You can remove the links at the bottom of the README too

**Comment by OskarStark** [comment]
Thanks

File: templates/project/README.md.twig (hunk 1 above)
Comment (VincentLanglet) [reply templates/project/README.md.twig:24]: Done, and I found some occurence of `project.stableBranch` so I added some checks.

**Review by phansys — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Can you please rebase? Thanks

File: templates/project/.github/PULL_REQUEST_TEMPLATE.md.twig [hunk 2]
```diff
@@ -3,13 +3,15 @@
...
+{% if project.stableBranch is not null %}
 <!--
     Show us you choose the right branch.
     Different branches are used for different things :
     - {{ project.stableBranch.name }} is for everything backwards compatible, like patches, features and deprecation notices
     - {{ project.unstableBranch.name }} is for deprecation removals and other changes that cannot be done without a BC-break
     More details here: https://github.com/sonata-project/{{ project.repository.name }}/blob/{{ project.stableBranch.name }}/CONTRIBUTING.md#base-branch
 -->
+{% endif %}
 I am targeting this branch, because {reason}.
```
Comment (core23) [inline templates/project/.github/PULL_REQUEST_TEMPLATE.md.twig:15]: Can we move this line inside the `stableBranch` block?

Comment (VincentLanglet) [reply templates/project/.github/PULL_REQUEST_TEMPLATE.md.twig:14]: Done

**Pull request merged**